web/admin/application/configs/klear/ProxyTrunksList.yaml

```diff
@@ -21,23 +21,66 @@ production:
           title: _("Options")
           screens:
             proxyTrunksEdit_screen: true
-          default: proxyTrunksEdit_screen
-    proxyTrunksEdit_screen: &proxyTrunksEdit_screenLink
+            proxyTrunksEditMain_screen: true
+          default: false
+          dialogs:
+            proxyTrunksDel_dialog: true
+      options:
+        title: _("Options")
+        screens:
+          proxyTrunksNew_screen: true
+    proxyTrunksEdit_screen:
       <<: *ProxyTrunks
       controller: edit
       class: ui-silk-pencil
+      parentOptionCustomizer:
+      - IvozProvider_Klear_Options_OptionsCustomizer
       label: false
       title: _("Edit %s %2s", ngettext('Proxy Trunks', 'Proxies Trunks', 1), "[format| (%item%)]")
-      fixedPositions:
+      fixedPositions: &proxyTrunksFixedPositionsLink
         group0:
           label: _("Basic Configuration")
           colsPerRow: 3
           fields:
             name: 1
             ip: 1
+    proxyTrunksEditMain_screen:
+      <<: *ProxyTrunks
+      controller: edit
+      class: ui-silk-pencil
+      parentOptionCustomizer:
+      - IvozProvider_Klear_Options_OptionsCustomizer
+      label: false
+      title: _("Edit %s %2s", ngettext('Proxy Trunks', 'Proxies Trunks', 1), "[format| (%item%)]")
```

Comment:
The other labels (Add and Delete) are _Proxy Trunks_ and plural _Proxy Trunks_
This one is _Proxy Trunks_ and plural _Proxies Trunks_